airbyte-integrations/connectors/source-google-search-console/source_google_search_console/manifest.yaml

```diff
@@ -605,8 +605,8 @@ definitions:
         request_headers:
           Content-Type: "application/json"
         request_body_json:
-          startDate: "{{ config.get('start_date') }}"
-          endDate: "{{ config.get('end_date') }}"
+          startDate: "{{ config.get('start_date', '2021-01-01') }}"
+          endDate: "{{ config.get('end_date', today_utc()) }}"
```

Comment:
unrelated to permissions, but given that start date has a default, but no strict required, its possible for customers to not set this